website/docs/plugins/basics.html.md

```diff
@@ -8,14 +8,16 @@ description: |-
 
 # Plugin Basics
 
-This page documents the basics of how the plugin system in Terraform
-works, and how to setup a basic development environment for plugin development
-if you're writing a Terraform plugin.
-
 ~> **Advanced topic!** Plugin development is a highly advanced
 topic in Terraform, and is not required knowledge for day-to-day usage.
 If you don't plan on writing any plugins, we recommend not reading
-this section of the documentation.
```

Comment:
s/we recommend not reading this section of the documentation/this section of the documentation is not necessary to read.